--- a/src/video_core/pica_types.h	2026-09-30 05:40:18.463371698 +0000
+++ b/src/video_core/pica_types.h	2026-09-30 05:40:18.465307457 +0000
@@ -36,7 +36,7 @@
 
         Float<M, E> res;
 
-#ifdef __FLT16_MANT_DIG__
+#if defined(__FLT16_MANT_DIG__) && (defined(__clang__) || __GNUC__ >= 13)
         if constexpr (M == 10 && E == 5) {
             res.value = std::bit_cast<_Float16>(static_cast<u16>(hex));
             return res;
